packages/statemanager/test/statelessVerkleStateManager.spec.ts

```diff
@@ -43,4 +52,123 @@ describe('StatelessVerkleStateManager: Kaustinen Verkle Block', () => {
       'should have correct codeHash'
     )
   })
+
+  it('put/delete/modify account', async () => {
+    const stateManager = new StatelessVerkleStateManager({ common })
+    stateManager.initVerkleExecutionWitness(block.executionWitness)
+
+    const address = new Address(randomBytes(20))
+
+    try {
+      await stateManager.getAccount(address)
+      assert.fail('should throw on getting account that is not found in witness')
+    } catch (e) {
+      assert.equal(
+        e.message.slice(0, 25),
+        'Missing execution witness',
+        'should throw on getting account that does not exist in cache and witness'
+      )
+    }
+
+    const account = Account.fromAccountData({
+      nonce: BigInt(2),
+    })
+    await stateManager.putAccount(address, account)
+
+    let check = await stateManager.getAccount(address)
+    assert.deepEqual(check, account, 'should return correct account')
+
+    await stateManager.modifyAccountFields(address, {
+      nonce: BigInt(2),
+    })
```

Comment:
I am not sure I understand the purpose of this `modifyAccountFields` and subsequent check (since we're modifying with the value it was already created with). Do we actually want to check that "modifying with the same value does not return a different account"? If so can we describe that, either in the assert description or in a comment?

Reply:
I was actually testing to see if updating by using `modifyAccountFields` is working as expected by updating an arbitrary account field. I have fixed it.